.circleci/config.yml

```diff
@@ -106,7 +106,7 @@ executors:
       # Set cluster workers to 5 (5 cluster nodes)
       # NUM_NODES represent number of parallel test executions
       # NUM_WORKERS represent number of nodes in a k8 cluster
-      NUM_NODES: 5 
+      NUM_NODES: 3 
```

Comment:
Nit: Is this done for a specific purpose?

Reply:
We want to reduce the number of test cases that run in parallel.